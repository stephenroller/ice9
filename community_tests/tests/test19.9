# write ? true * false; write ? true * true;		# boolean and

write ? (true * false); 
write ? (true * true);

write ? (false * true);
write ? (false * false);
