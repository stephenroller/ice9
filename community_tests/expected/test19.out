Loading file: generated/test19.tm
0 
1 
0 
0 
Number of instructions executed = 45
